lib/Doctrine/ODM/PHPCR/Query/Builder/QueryBuilder.php

```diff
@@ -80,7 +81,7 @@ public function getQuery()
     /**
      * @param BuilderConverterPhpcr $converter
      */
-    public function setConverter(BuilderConverterPhpcr $converter)
+    public function setConverter(BuilderConverterBase $converter)
```

Comment:
is there a chance that people extend the QueryBuilder? if they also overwrite this method, this would be a BC break. but i guess the risk for that is really small. current stable is 1.2 but master is 1.3 so that is ok imho